### PR TITLE
Fix #20 (whitespace related render issues)

### LINKED ISSUE #20: Spinner is broken because of space between div tags
Hello everyone!
I'm testing paper-spinner on my application in ruby on rails, and it broke if a space is putted between a specific closing and opening div.

this is the original
```html
<div class="spinner-layer layer-1">
        <div class="circle-clipper left">
          <div class="circle"></div>
        </div><div class="gap-patch">
          <div class="circle"></div>
        </div><div class="circle-clipper right">
          <div class="circle"></div>
        </div>
      </div>
```

the div is .gap-patch

```html
<div class="spinner-layer layer-1">
        <div class="circle-clipper left">
          <div class="circle"></div>
        </div> <div class="gap-patch">
          <div class="circle"></div>
        </div><div class="circle-clipper right">
          <div class="circle"></div>
        </div>
      </div>
```
even if I put the original code to be referenced in my application the ruby on rails break the lines in the node http-server that does not occur

### PR DESCRIPTION
**Pull request by max96at**

### CODE REVIEW

**Comment by max96at** [comment]
I signed it!

File: paper-spinner.css [hunk 1]
```diff
@@ -64,6 +64,11 @@ subject to an additional IP rights grant found at http://polymer.github.io/PATEN
   opacity: 0;
 }
 
+/* Fix for whitespace related render issues */
+.spinner-layer > div {
+  float: left;
```
Comment (notwaldorf) [inline paper-spinner.css]: This will break RTL, unfortunately

**Comment by keanulee** [comment]
@max96at Is there a reason why you need to do this? Please see my comment on issue #20 at https://github.com/PolymerElements/paper-spinner/issues/20#issuecomment-158185474

**Comment by keanulee** [comment]
Closing due to inactivity

**Pull request closed without merging**